src/timesync/timesyncd-manager.c

```diff
@@ -1123,7 +1127,9 @@ int manager_new(Manager **ret) {
         if (r < 0)
                 return r;
 
-        manager_network_read_link_servers(m);
+        r = manager_network_read_link_servers(m);
```

Comment:
i don't think this should be fatal either, if we can't read the interface data, that should be mostly OK. That said, the current code is a bit sloppy: we really should log about this case, and not just eat it up silently...

Reply:
Indeed. I've dropped the change here, and added more log messages in `manager_network_read_link_servers()`.